arch/sim/src/sim/up_internal.h

```diff
@@ -32,7 +32,9 @@
 #ifndef __ASSEMBLY__
 #  include <sys/types.h>
 #  include <stdbool.h>
-#  include <netinet/in.h>
+#  if defined(CONFIG_SIM_NETDEV_TAP) || defined(CONFIG_SIM_NETDEV_VPNKIT)
```

Comment:
let's remove the empty vpnkit_ifup and vpnkit_ifdown, adn remove CONFIG_SIM_NETDEV_VPNKIT